.github/workflows/tag_and_publish.yaml

```diff
@@ -0,0 +1,43 @@
+name: Tag and Publish Release
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.ref }}
+
+jobs:
+  tag-release:
+    runs-on: ubuntu-18.04
+    outputs:
+      version: ${{ steps.tag-release.outputs.version}}
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+      - name: Tag release
+        id: tag-release
+        run: |
+          version=$(cat VERSION)
+          git tag $version
```

Comment:
Do we need to check for major version bumps and drive annotated tags so `git describe` better represents major or major.minor version bumps?

Reply:
I'm not really sure the use case for `git describe` on pycloudlib. What's the value in seeing how many commits there have been since the last major or minor version bump?

Reply:
Good thought. I suppose in this CD world we are bumping major, minor or subrevision with each merged commit anyway.... So, that commit count offset  is encoded in the subrevision basically.